lib/Doctrine/Common/Annotations/DocParser.php

```diff
@@ -426,7 +426,7 @@ private function match(int $token): bool
      * If any of them matches, this method updates the lookahead token; otherwise
      * a syntax error is raised.
      *
-     * @phpstan-param list<mixed[]> $tokens
+     * @phpstan-param list<int|string> $tokens
```

Comment:
This seems to be a phpdoc fix for the existing code, unrelated to supporting enums. So I suggest submitting it as a separate PR instead.

Reply:
Fixed